### PR TITLE
[MM-30175] Migrate changeCSS() to CSS variable in utils/utils.jsx, ln. 709

### PR DESCRIPTION
**Pull request by IanvsPoplicola**
#### Summary
Migrate a CSS variable for popover from `utils/utils.jsx` into `sass/components/_popover.scss`

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-server/issues/16179

### CODE REVIEW

File: sass/components/_popover.scss [hunk 1]
```diff
@@ -68,7 +68,13 @@
 
     &.bottom,
     &.right,
-    &.top,
+
+    &.top {
```
Comment (hmhealey) [inline sass/components/_popover.scss:72]: Same comment as [here](https://github.com/mattermost/mattermost-webapp/pull/6993#discussion_r516254990)

**Review by hmhealey — APPROVED** [review]

**Review by calebroseland — APPROVED** [review]
LGTM

**Review by deanwhillier — APPROVED** [review]
LGTM, thanks so much @IanvsPoplicola!

**Comment by deanwhillier** [comment]
Haha, thanks Caleb! 😂

**Comment by calebroseland** [comment]
Np! been sitting on these for too long!

**Pull request merged**